### PR TITLE
[PL-47660] DEL 82402 Release Notes

### PR DESCRIPTION
**Pull request by saifeemustafaqharness**
[PL-47660](https://harness.atlassian.net/browse/PL-47660)

https://harness.atlassian.net/wiki/spaces/PD/pages/21663350825/NG+Release+SAAS+February+19th+2024+for+1.26.x-+Build

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screenshot. 

[PL-47660]: https://harness.atlassian.net/browse/PL-47660?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: release-notes/delegate.md [hunk 1]
```diff
@@ -40,6 +40,29 @@ import Deleos from '/docs/platform/shared/delegate-legacy-eos.md'
...
 ## February 2024
 
+### Harness version 1.26.8, Harness Delegate version 24.02.82400 <!--  February 26, 2024 -->
+
+#### Fixed issues
+
+- The retry interval for attempting to create or read secrets from HashiCorp Vault was fixed at 1 second after each failure. (PL-46595, ZD-57053)
+   The retry interval has now been modified to increase by a factor of 2 times the number of failures. Consequently, after the first failure, the second attempt will occur after a 2-second delay, and the third attempt will be made after a 4-second delay, enhancing the robustness of secret management operations.
+
+- When linking an SSO group with over 1,000 users, only 1,000 users were syncing in Harness due to a limitation with LDAP groups syncing. (PL-46492)
```
Comment (brian-f-harness) [inline release-notes/delegate.md:50]:
```suggestion
- When linking an SSO group with over 1,000 users, only 1,000 users were syncing in Harness due to a limitation with LDAP groups syncing. (PL-46492, ZD-56741)
```

**Review by brian-f-harness — COMMENTED** [review]
Suggest adding the three items published with 1.25.5 (PL-46595, PL-46492, and PL-42600) that are on the 1.26.x list for the delegate release. 1.25.5 wasn't a delegate release week, and they have the Affects-delegate label.

**Review by brian-f-harness — APPROVED** [review]
LGTM

**Review by pratmit — APPROVED** [review]
LGTM

**Pull request merged**